src/Core/src/Platform/Android/EditTextExtensions.cs

```diff
@@ -235,7 +235,10 @@ static void UpdateCursorSelection(EditText editText, ITextInput entry)
 				int start = GetSelectionStart(editText, entry);
 				int end = GetSelectionEnd(editText, entry, start);
 
-				editText.SetSelection(start, end);
+				if (editText.IsFocused)
+					editText.Post(() => editText.SetSelection(start, end));
```

Comment:
I know this is probably overkill, but maybe this should be:

```suggestion
					editText.Post(() => 
					{
						if (editText.IsAlive())
						{
							editText.SetSelection(start, end);
						}
					);
```

Reply:
Sure! Good idea